source/plugin/permissions.rst

```diff
@@ -153,9 +153,9 @@ Otherwise it will be inherited from any parent ``Subject``. It does not matter w
 
 If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
 default ``Subjects``. Each ``SubjectCollection`` defines its own defaults. The global and weakest default subject can be
-obtained from the ``PermissionService``. It is recommended that plugins define their own permissions to the default's
-transient ``SubjectData`` during every server start-up. This allows the server owner to overwrite the defaults defined
-by plugins according to his needs using the default's persistent ``SubjectData``.
+obtained from the ``PermissionService``. It is recommended (but not mandatory) that plugins define their own permissions
```

Comment:
It's a case of whether we want to encourage plugins that work "out of the box" or plugins where some work has to go into configuring their permissions.

I think it's better that you have to explicitly give each permission; it avoids a situation where your players get automatically given access to something you weren't even aware of. It also forces people to read documentation, which is only a good thing.

However, if that's the stance we want to take, then what's the point in having default subjects in the first place?

I'm not really sure how I feel. I can see the benefits and drawbacks of each approach.